.github/workflows/release.yml

```diff
@@ -17,6 +17,10 @@ on:
           - 'patch' # bug fixes
           - 'minor' # new features, backwards compatible
           - 'major' # breaking changes
+          - 'prepatch' # bumps the patch and moves to a prerelease (1.0.2 -> 1.0.3a0)
+          - 'preminor' # bumps the minor and moves to a prerelease (1.0.2 -> 1.1.0a0)
+          - 'premajor' # bumps the major and moves to a prerelease (1.0.2 -> 2.0.0a0)
+          - 'prerelease' # bumps the prerelease version (1.0.3a0 -> 1.0.3a1)
```

Comment:
Theoretically, you also need to pass `prerelease: True` to the "Create GH release" step.  
(Not critical, I guess)

Reply:
Good point fixed!